tests/test_parquet.py

```diff
@@ -138,6 +139,7 @@ def _makeSimpleNumpyTable(include_multidim=False, include_bigendian=False):
         ("f", "i8"),
         ("strcol", "U10"),
         ("bytecol", "a10"),
+        ("dt", "datetime64[ns]"),
```

Comment:
Can you add to the test the `us` as well? Is it relevant?

Reply:
I will add that!